### PR TITLE
add shortcut to run shfmt locally

### PR DESCRIPTION
**Pull request by itscaro**
maybe?

### CODE REVIEW

**Comment by mlocati** [comment]
Users don't know what `run the lint to reformat changes` means. Furthermore this isn't portable (the makefile don't work on Windows)

**Comment by itscaro** [comment]
> Users don't know what `run the lint to reformat changes` means. Furthermore this isn't portable (the makefile don't work on Windows)

I use busybox on windows

**Comment by mlocati** [comment]
I use WSL, but still there's no explanation about what users should do when you say "run the lint to reformat changes"

**Comment by itscaro** [comment]
@mlocati I tried to re-formulate the phrase, and for Makefile, I use a compiled version on Windows (not busybox, sorry)

File: Dockerfile-shfmt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM golang
+
+WORKDIR /src
+
+RUN go get github.com/mvdan/sh/cmd/shfmt
+
+#COPY install-php-extensions .
+#COPY scripts scripts
+
+#RUN shfmt -d -s -ln posix -i 0 -ci -kp install-php-extensions scripts/common scripts/travisci-test-extensions scripts/travisci-update-readme scripts/update-readme
```
Comment (itscaro) [inline Dockerfile-shfmt]: Useless for now, copied from travis-ci.yml

**Comment by mlocati** [comment]
What about https://github.com/mlocati/docker-php-extension-installer/pull/79?

**Comment by mlocati** [comment]
Superseded by #79

**Pull request closed without merging**